subprojects/docs/src/snippets/kotlinDsl/multiProjectBuild/kotlin/settings.gradle.kts

```diff
@@ -5,6 +5,12 @@ pluginManagement {
         gradlePluginPortal()
     }
 }
+
```

Comment:
This will show in example 9 for this section: https://docs.gradle.org/nightly/userguide/kotlin_dsl.html#sec:multi_project_builds_applying_plugins

It would be distracting. We could either leave it outside of the `tag:repositories` section, or have `repositories { mavenCentral() }` declared for each project required. Note we already do that for the `http` subproject.

Reply:
Done!